src/main/java/org/springframework/social/salesforce/api/impl/ConnectTemplate.java

```diff
@@ -37,9 +42,19 @@ public ConnectTemplate(Salesforce api, RestTemplate restTemplate) {
 
         requireAuthorization();
 
-        JsonNode dataNode = restTemplate.getForObject(api.getBaseUrl() + "/" + getVersion() + "/connect/communities", JsonNode.class, getVersion());
+        JsonNode dataNode = restTemplate.getForObject(api.getBaseUrl() + "/" + getVersion() + CONNECT_URI, JsonNode.class, getVersion());
 
         return api.readList(dataNode.get("communities"), Community.class);
     }
 
+    @Override
+    public List<CommunityUser> getCommunityUsers(String communityId) {
+
+        requireAuthorization();
+
+        JsonNode dataNode = restTemplate.getForObject(api.getBaseUrl() + "/" + getVersion() + CONNECT_URI + communityId + CHATTER_USERS, JsonNode.class, getVersion());
```

Comment:
@jottley what do you think about this? Basically the call to the **/communities/communityID/chatter/users/userID** SF endpoint won't give us the account to which the user is attached. Should we get that info from another REST call? For example a query?

Reply:
Correct. Here we just want to return the results from the call to Salesforce.  We allow the "customer" to decide in their project what to do with that info and if there are multiple calls needed.